samples/react-timeline/src/webparts/timeline/components/Popup/timelineEvent.module.scss

```diff
@@ -0,0 +1,55 @@
+@import "~@microsoft/sp-office-ui-fabric-core/dist/sass/SPFabricCore.scss";
+
+:global(.ms-Dialog-inner) {
```

Comment:
Strongly recommend that you wrap the `:global` tag within your timelineevent CSS class -- otherwise you're affecting EVERY web part on the page. That isn't good CSS behavior for any web part.